src/main/java/de/tum/cit/aet/artemis/iris/domain/settings/IrisExerciseSettings.java

```diff
@@ -69,6 +69,16 @@ public void setIrisTextExerciseChatSettings(IrisTextExerciseChatSubSettings iris
         this.irisTextExerciseChatSettings = irisTextExerciseChatSettings;
     }
 
+    @Override
+    public IrisCourseChatSubSettings getIrisCourseChatSettings() {
+        return null;
+    }
+
+    @Override
+    public void setIrisCourseChatSettings(IrisCourseChatSubSettings irisCourseChatSettings) {
+
+    }
```

Comment:
Let's add a short comment in the boy of this method to justify that it's empty.

```suggestion
    public void setIrisCourseChatSettings(IrisCourseChatSubSettings irisCourseChatSettings) {
        // Empty because exercises don't have course chat settings
    }
```